Dockerfile

```diff
@@ -47,12 +47,14 @@ RUN pip install pipenv \
     cspell \
     jscpd \
     markdown-link-check \
-    && mkdir -p /opt/goat/log
+    && mkdir -p /opt/goat/log \
+    # The following commands are necessary because pre-commit adds -u os.uid():os.gid() to the docker run
+    && chmod o+w /opt/goat/log \
+    && mkdir -p /.local \
```

Comment:
This fixes the following error:
```bash
pipenv run mypy --config-file /etc/opt/goat/.mypy.ini --install-types --non-interactive ./tasks.py
Defaulting to user installation because normal site-packages is not writeable
Collecting types-requests
  Downloading types_requests-2.31.0.1-py3-none-any.whl (14 kB)
Collecting types-urllib3
  Downloading types_urllib3-1.26.25.13-py3-none-any.whl (15 kB)
Installing collected packages: types-urllib3, types-requests
ERROR: Could not install packages due to an OSError: [Errno 13] Permission denied: '/.local'
Check the permissions.
```

Reply:
Specifically note this line:
Defaulting to user installation because normal site-packages is not writeable